app/src/main/java/it/quip/android/view/RoundedImageTransform.java

```diff
@@ -0,0 +1,45 @@
+package it.quip.android.view;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapShader;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.RectF;
+import android.graphics.Shader;
+
+public class RoundedImageTransform implements com.squareup.picasso.Transformation {
+
```

Comment:
I think we should play around with this in polish week and see if we can get a nice unique look, but thats #laterthangs